test/conformance/testdata/conformance.txt

```diff
@@ -106,34 +106,33 @@ test/e2e/common/pods.go: "should be submitted and removed"
 test/e2e/common/pods.go: "should be updated"
 test/e2e/common/pods.go: "should allow activeDeadlineSeconds to be updated"
 test/e2e/common/pods.go: "should contain environment variables for services"
-test/e2e/common/projected.go: "should be consumable from pods in volume"
-test/e2e/common/projected.go: "should be consumable from pods in volume with defaultMode set"
-test/e2e/common/projected.go: "should be consumable from pods in volume as non-root with defaultMode and fsGroup set"
-test/e2e/common/projected.go: "should be consumable from pods in volume with mappings"
-test/e2e/common/projected.go: "should be consumable from pods in volume with mappings and Item Mode set"
-test/e2e/common/projected.go: "should be consumable in multiple volumes in a pod"
-test/e2e/common/projected.go: "optional updates should be reflected in volume"
-test/e2e/common/projected.go: "should be consumable from pods in volume"
-test/e2e/common/projected.go: "should be consumable from pods in volume with defaultMode set"
-test/e2e/common/projected.go: "should be consumable from pods in volume as non-root"
-test/e2e/common/projected.go: "should be consumable from pods in volume with mappings"
-test/e2e/common/projected.go: "should be consumable from pods in volume with mappings and Item mode set"
-test/e2e/common/projected.go: "should be consumable from pods in volume with mappings as non-root"
-test/e2e/common/projected.go: "updates should be reflected in volume"
-test/e2e/common/projected.go: "optional updates should be reflected in volume"
-test/e2e/common/projected.go: "should be consumable in multiple volumes in the same pod"
-test/e2e/common/projected.go: "should provide podname only"
-test/e2e/common/projected.go: "should set DefaultMode on files"
-test/e2e/common/projected.go: "should set mode on item file"
-test/e2e/common/projected.go: "should update labels on modification"
-test/e2e/common/projected.go: "should update annotations on modification"
-test/e2e/common/projected.go: "should provide container's cpu limit"
-test/e2e/common/projected.go: "should provide container's memory limit"
-test/e2e/common/projected.go: "should provide container's cpu request"
-test/e2e/common/projected.go: "should provide container's memory request"
-test/e2e/common/projected.go: "should provide node allocatable (cpu) as default cpu limit if the limit is not set"
-test/e2e/common/projected.go: "should provide node allocatable (memory) as default memory limit if the limit is not set"
 test/e2e/common/projected.go: "should project all components that make up the projection API"
```

Comment:
Why wasn't this one moved?

Reply:
This tests projected secrets, configmaps and downward API combined in one pod.